### ISSUE TITLE
/usr/sbin/ufw

### ISSUE THREAD

**Issue by lonesomewalker** [bigbluebutton/bigbluebutton#10922]
After upgrading to 2.2.30, bbb-conf seems to havoc:

```
root@bbbserver ~ # bbb-conf --restart
/usr/sbin/ufw
Restarting BigBlueButton 2.2.30 ...
Stopping BigBlueButton
Starting BigBlueButton



# Potential problems described below
/usr/sbin/ufw
......
root@bbbserver ~ #
```

ufw is installed, also rules are applied.

This is kind of strange, what messed up from 2.2.29 to 2.2.30?
I mean, also the notify of using a firewall and TURN server...
Neccessary? Really?

**Comment by Dennis1993** [comment]
I've the same message after upgrading BBB from 2.2.29 to 2.2.30

**Comment by ffdixon** [comment]
The fix for this will be in 2.2.31.  

  https://github.com/bigbluebutton/bigbluebutton/pull/10920

**Comment by ho30hero** [comment]
> The fix for this will be in 2.2.31.
> 
> #10920

How can we find fast that the new stable version released ?

**Comment by lonesomewalker** [comment]
Visit: https://github.com/bigbluebutton/bigbluebutton
Look at the right side. There is written the latest release number.
